src/Common/Commands.Common.Authentication.Abstractions/Interfaces/IAuthenticationFactory.cs

```diff
@@ -35,7 +35,7 @@ public interface IAuthenticationFactory : IHyakAuthenticationFactory
         /// <param name="tokenCache">Token Cache</param>
         /// <param name="resourceId">Optional, the AD resource id</param>
         /// <returns></returns>
-        IAccessToken Authenticate(
+        IRenewableToken Authenticate(
```

Comment:
This needs to continue to return IAccessToken, otherwise this is a breaking change

Reply:
Done